app/templates/components/edit-analysis-details.emblem

```diff
@@ -191,9 +191,9 @@
         .column
           form
             p.control.is-fullwidth
-              = file-upload name="files" accept="*/*" onfileadd=(action "uploadFile") as |queue|
-                a class="button is-primary" disabled=isUploading
-                  if isUploading
+              = file-upload name="files" accept="*/*" onfileadd=( action 'uploadFileWrapper' ) as |queue0|
```

Comment:
queue0?

Reply:
@cosmosgenius  will fix all :+1: , this one I think gets added by mistake during committing the changes, thanks for the review  :+1: